usaspending_api/database_scripts/matview_generator/subaward_view.json

```diff
@@ -93,15 +93,17 @@
     "LEFT OUTER JOIN",
     "  (SELECT",
     "    legal_business_name AS recipient_name,",
-    "    duns",
+    "    duns,",
+    "    ROW_NUMBER() OVER(PARTITION BY duns) AS row",
     "  FROM recipient_lookup AS rlv",
-    "  ) recipient_lookup ON recipient_lookup.duns = recipient_unique_id AND recipient_unique_id IS NOT NULL",
+    "  ) recipient_lookup ON recipient_lookup.duns = recipient_unique_id AND recipient_unique_id IS NOT NULL AND row = 1",
```

Comment:
Is it safe to continue to join to `recipient_lookup` based on `duns`? When we stop receiving transactions with duns, we'll need something else. `uei` or `recipient_hash`.

Reply:
We currently do not have UEI data for subawards in the usaspending databases, when we get that data we can update the subaward view to use those, but for now this is what it is.

Reply:
I'll create a ticket to load `uei` from the broker `subaward` table into USAS. We can update this at that time.